website/content/projects/2019/presentation.md

```diff
@@ -30,18 +30,26 @@ You do not need to dress formally for the presentations. Casual clothing is more
 
 ## Scoring
 
-The presentation will be worth 10 points of your final grade. 6 of these points will be determined by the instructor and TAs, and the remaining 4 points will be determined by a panel of guest judges. Both the instructor/TAs and the guest judges will score your presentation (on a scale of 0 to 5) on two aspects: Clarity/structure (weighed 75%) and Q&A (weighed 25%).
+The presentation will be worth 10 points of your final grade. 6 of these points will be determined by the instructor and TAs, and the remaining 4 points will be determined by a panel of guest judges. The  instructor/TAs will score your presentation (on a scale of 0 to 5) on three aspects: Clarity/structure (weighed 40%), demo (weighed 35%) and Q&A (weighed 25%). The guest judges will score your presentation only on clarity/structure (75%) and Q&A (25%).
```

Comment:
Why are the guest judges not grading the demo?

Reply:
The guest judges are not as familiar with the expectations of the project and the demo, and it is unlikely they'll be able to adequately judge whether a demo is actually highlighting what a team did (unlike the instructors/TAs, who are much more familiar with the internals of the project, and the expectations of the demos) At the very least, students could legitimately complain that guest judges gave them a demo score that was not well-founded, which is harder to argue on clarity/structure and Q&A.